dist/kafka-sampler/config.yaml

```diff
@@ -13,16 +13,23 @@ kafka:
   # Convert field names to all lowercase and divide nested structs with `_`
   # sarama:  (...)
 
-  # If unset, it will create a `Sampler` per each topic found in the Kafka cluster,
-  # supports regex RE2 syntax as described at https://github.com/google/re2/wiki/Syntax, except for `\C`.
-  # It always ignores internal topics like: `__consumer_offsets` and `__transaction_state`.
-  # topicfilter: 
+  # Topics contains all the configuration about the topics that have to be read from
+  # topics:
+  #   # Maximum number of topics to read from (safeguard). Topics after the limit is reached
+  #   # are ignored
+  #   max: 25
+  #
   #   # Topic list refresh period. In each refresh it will create/delete `Samplers` based on the cluster existing topics
   #   refreshperiod: 1m
-  #   # Topics matching `allowlist` will be monitored, and topics matching `denylist` will be ignored.
-  #   # `allowlist` and `denylist` can't be set at the same time.
-  #   allowlist:
-  #   denylist:
+  #
+  #   # If unset, it will create a `Sampler` per each topic found in the Kafka cluster,
+  #   # supports regex RE2 syntax as described at https://github.com/google/re2/wiki/Syntax, except for `\C`.
+  #   # It always ignores internal topics like: `__consumer_offsets` and `__transaction_state`.
+  #   topicfilter:
```

Comment:
this jey has been renamed to `filter`, right?

Reply:
Yes, done.